elements/bulbs-slideshow/bulbs-slideshow.js

```diff
@@ -103,9 +107,31 @@ class BulbsSlideshow extends BulbsHTMLElement {
 
   afterSlideTransition () {
     this.render();
+    this.sendPageView();
     this.navigateToNextSlide();
   }
 
+  setupGA () {
+    const readingListProps = prepReadingListAnalytics(this.$slideshow, { dimension12: 'slideshow' });
+    const { analyticsManager, analyticsWrapper, title } = readingListProps;
+
+    this.analyticsManager = analyticsManager;
+    this.analyticsWrapper = analyticsWrapper;
+    this.title = title;
+  }
+
+  sendPageView () {
```

Comment:
so @spra85 if you could give this another pair of 👀, I dug into this and was able to confirm the following:
 - this event (`sendPageView`) _always_ fires when clicking next/previous
   - subsequently, `trackPageView` always fires
 - the only `ga('send','pageview')` events that the GA debugger picks up are those _after_ the 1st time 'next' is clicked
    - there's no code that specifies that the event shouldn't fire on the first click; everything 
 needed exists. 
    - and the code is (seemingly) identical to clickventure's (which fires on 1st 'next')